files/deployable/index.js

```diff
@@ -2,6 +2,10 @@ const { IAMClient, GetRolePolicyCommand } = require('@aws-sdk/client-iam');
 const { SSMClient, GetParameterCommand } = require('@aws-sdk/client-ssm');
 const { STSClient, GetCallerIdentityCommand } = require('@aws-sdk/client-sts');
 
+const fs = require('fs');
+// Check if the file exists
+const configFile = './config.json';
```

Comment:
move this configFile declaration to be alongside the other global variables so its not polluting the imports.